frontend/components/forms/fields/SelectTargetsDropdown/TargetOption/_styles.scss

```diff
@@ -24,6 +24,12 @@
     padding: 0;
   }
 
+  &__add-btn {
+    min-width: 60px;
+    display: flex !important;
+    justify-content: end !important;
+  }
+
```

Comment:
Any way we can not use `!important;` here? Add more specificity in the selector, or reduce the specificity of something else?

Reply:
I took a second look. I realized I didn't need `display: flex` since `inline-flex` was already in play. But for `justify-content`, it's already set at the lowest level for the `button` class so it has to be overwritten. If it's any consolation, this component will be removed in favor of `components/TargetsInput`.

Reply:
Got it. If we're replacing it soon I'm okay with the `!important`.